go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	github.com/ONSdigital/dp-elasticsearch/v3 v3.0.0-alpha.7 // indirect
 	github.com/ONSdigital/dp-mongodb-in-memory v1.3.1 // indirect
 	github.com/ONSdigital/dp-rchttp v1.0.0 // indirect
-	github.com/ONSdigital/dp-search-api v1.29.0 // indirect
+	github.com/ONSdigital/dp-search-api v1.7.1-0.20230120122810-91b330fb217d // indirect
```

Comment:
We look to only add dependencies of our tagged releases

So will need to get dp-search-api released and tagged and then amend the go.mod file to point to new version here

Reply:
Fixed now